DependencyInjection/eZDemoCompilerPass.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+class eZDemoCompilerPass implements CompilerPassInterface
```

Comment:
The easiest way is to use service aliases, usually.

In the PrivacyCookieBundle, the factory service, `ez_privacy_cookie.banner_factory`, is an alias for `ez_privacy_cookie.configuration_banner_factory`. In the DemoBundle, just redeclare this service to be an alias of yours.

And then you should not need the compiler pass anymore.